### PR TITLE
feat: v-on in component

### PR DESCRIPTION
**Pull request by Jevon617**

### CODE REVIEW

File: packages/runtime-vapor/src/index.ts [hunk 1]
```diff
@@ -123,6 +123,8 @@ export { createComponent } from './apiCreateComponent'
 
 export { resolveComponent, resolveDirective } from './helpers/resolveAssets'
 
+export { toHandlers } from './helpers/toHandlers'
```
Comment (sxzz) [inline packages/runtime-vapor/src/index.ts:126]: Revert this

File: packages/runtime-vapor/src/helpers/toHandlers.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { isObject, toHandlerKey } from '@vue/shared'
```
Comment (sxzz) [inline packages/runtime-vapor/src/helpers/toHandlers.ts:1]: Not necessary

**Review by sxzz — COMMENTED** [review]
Compiler part is done via 9e0cd20da02b8a84cf780065367840fcd1947bfb

You can continue work on runtime part. Thanks anyway!

**Comment by Jevon617** [comment]
> Compiler part is done via [9e0cd20](https://github.com/vuejs/core-vapor/commit/9e0cd20da02b8a84cf780065367840fcd1947bfb)
> 
> You can continue work on runtime part. Thanks anyway!

Got it, I'll proceed with the runtime part. Thanks for letting me know!

**Pull request closed without merging**